testsuite/integration-arquillian/servers/auth-server/quarkus/pom.xml

```diff
@@ -335,7 +335,44 @@
 
                 </plugins>
             </build>
+        </profile>
 
+        <profile>
+            <id>product</id>
```

Comment:
Works as expected. The Oracle JDBC driver is included in the `/providers` dir only when the `product` profile is activated.

Reply:
This doesn't do anything really as we test DB support with GitHub Actions which doesn't use the `product` profile. Also not sure we should always install the Oracle Driver, but rather only install it when we are testing Oracle.